README.md

```diff
@@ -121,7 +121,7 @@ export default function (e, t, r) {
 {
 	"main": "./dist/foo.umd.js", // legacy UMD output (for Node & CDN use)
 	"module": "./dist/foo.module.js", // legacy ES Modules output (for bundlers)
```

Comment:
```suggestion
	"module": "./dist/foo.module.mjs", // legacy ES Modules output (for bundlers)
```

Reply:
whoops, good catch.

Reply:
Nothing like an approval to make mistakes stand out 😅